app/d3plus.js

```diff
@@ -187,6 +259,17 @@ export default {
     }
   },
   legendPosition: "bottom",
+  legendTooltip: {
+    title(d) {
+      const {item, itemId, parent, parentId} = getTooltipTitle(this, d);
+      const aggregated = Array.isArray(parent) ? "Valores" : parent;
+      const title = Array.isArray(item) ? `Otros ${aggregated || "Valores"}` : item;
+      const itemBgImg = parentId;
+      let bgColor = findColorTooltip(itemBgImg, d);
+      let imgUrl = findIconV2(itemBgImg, d);
+      return tooltipTitle(bgColor, imgUrl, title, parentId);
+    }
+  },
```

Comment:
From what I can see, `legendTooltip.title` and `tooltipConfig.title` are the exact same function, and they both use `getTooltipTitle` and `tooltipTitle`...

Can we collapse all 3 of these functions into one large `findTooltipTitle` function?